clab/config/template.go

```diff
@@ -43,10 +47,11 @@ func LoadTemplates(tmpl *template.Template, role string) error {
 }
 
 func RenderAll(nodes map[string]nodes.Node, links map[int]*types.Link) (map[string]*NodeConfig, error) {
+	// A map with the ShortName as the key
 	res := make(map[string]*NodeConfig)
 
-	if len(TemplatePaths) == 0 {
-		return nil, fmt.Errorf("please specify one of more paths with --template-path")
+	if len(TemplatePaths) == 0 { // default is the install path
+		TemplatePaths = []string{"@"}
```

Comment:
the `@` char is used here and not the absolute path or a string const because you want users to be able to pass a list of template paths like `@, my-path, my-other-path`?

Reply:
Indeed yes, just a shortcut to add the built-in templates to the path list